tool/generate_converter

```diff
@@ -48,7 +48,7 @@ done
 
 # Classes.
 for type in $(echo "$source" \
-  | egrep '^[a-z ]*class ' \
+  | grep -E '^[a-z ]*class ' \
```

Comment:
I'm using linux and this is what I get when I run fgrep/egrep

```
❯ egrep
egrep: warning: egrep is obsolescent; using grep -E
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.


❯ fgrep
fgrep: warning: fgrep is obsolescent; using grep -F
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
```


I did look up if there's any difference between the two and there seems to be none so I took the liberty to change it to the newer version